pkg/agent/agent.go

```diff
@@ -267,6 +267,7 @@ func (a *Agent) bootstrap() error {
 			BaseSVID:       a.BaseSVID,
 			BaseSVIDKey:    a.baseSVIDKey,
 			BaseRegEntries: regEntries,
+			BaseSVIDPath:   a.getBaseSVIDPath(),
```

Comment:
perhaps this is better modeled as a pkg-level var?

Reply:
💯